src/core/SIPTransactions/UASInviteTransaction.cs

```diff
@@ -121,12 +121,12 @@ public new void SendFinalResponse(SIPResponse sipResponse)
         /// generated final response.
         /// </summary>
         /// <returns>A socket error with the result of the cancel.</returns>
-        public void CancelCall()
+        public void CancelCall(SIPRequest sipCancelRequest)
         {

```

Comment:
This breaks the public API surface and will affect library users. Can you make it optional?